libs/components/packages/src/schematics/migrations/update-7/fix-imports.ts

```diff
@@ -126,8 +178,8 @@ async function visitTypescriptFiles(
       `Looking for typescript files in project ${projectName}...`
     );
     tree.getDir(project.root).visit((filePath) => {
-      if (filePath.endsWith('.ts')) {
-        updateTypescriptImports(filePath, tree, context);
+      if (filePath.endsWith('.ts') && !filePath.includes('node_modules')) {
```

Comment:
You could use `project.sourceRoot` instead of `project.root` to get at the source files.